python/flexflow/keras/backend/__init__.py

```diff
@@ -16,6 +16,7 @@
 from __future__ import absolute_import
 from __future__ import print_function
 import sys
+from .backend_functions import batch_dot, BatchMatmul
```

Comment:
Is the current preference in the codebase absolute or relative imports? We should max the existing python style when possible